script.js

```diff
@@ -75,9 +90,37 @@ const list = {
 const datatable = {
     view: "datatable",
     id: "table",
-    data: small_film_set,
+    url: "data/data.js",
     autoConfig: true,
-    scroll: "y"
+    scroll: "y",
+    select: true,
+    hover: "datatable_hover",
+    on: {
+        onAfterSelect: dataToForm
+    },
+    columns: [
+        { id: "rank", header: "", width: 50, css: "gray" },
+        { id: "title", header: ["Film Title", { content: "textFilter" }], fillspace: true, sort: "string" },
+        { id: "year", header: ["Released", { content: "numberFilter" }], sort: "int" },
+        { id: "votes", header: ["Votes", { content: "textFilter" }], sort: "int" },
+        { id: "rating", header: ["Rating", { content: "textFilter" }], sort: "int" },
+        { id: "deleteIcon", header: "", template: "{common.trashIcon()}" }
+    ],
+    onClick: {
+        "wxi-trash": function (e, id) {
+            this.remove(id.row)
+        }
+    },
+    scheme: {
+        $init: function (obj) {
+            if (obj.votes.includes(",")) {
+                obj.votes = obj.votes.replace(",", ".") * 1000;
+            }
+            if (obj.rating.includes(",")) {
+                obj.rating = obj.rating.replace(",", ".");
```

Comment:
by this `replace` you actually decrease ratings by 1000. why not simply replace commas by empty strings?

Reply:
"All genius is simple", fixed it.